pkgs/applications/misc/xmr-stak/default.nix

```diff
@@ -18,11 +18,18 @@ stdenv.mkDerivation rec {
 
   NIX_CFLAGS_COMPILE = "-O3";
 
+  patches = [ (fetchpatch {
+    name = "fix-libmicrohttpd-0-9-71.patch";
+    url = "https://patch-diff.githubusercontent.com/raw/fireice-uk/xmr-stak/pull/2739.patch";
```

Comment:
It's uglier, but in cases where the PR isn't merged we tend to prefer commit id references because the contents of PRs can change. In this case, probably https://github.com/fireice-uk/xmr-stak/commit/78b433bac8226c39eb9b4aa099d7a713ddef4485.patch and https://github.com/fireice-uk/xmr-stak/commit/8adb208987f5881946992ab9cd9a45e4e2a4b870.patch